bpo-9938: Add optional keyword argument exit_on_error to argparse.ArgumentParser (#15362)
python/cpython: Doc/library/argparse.rst

   * allow_abbrev_ - Allows long options to be abbreviated if the
     abbreviation is unambiguous. (default: ``True``)

>    * exit_on_error_ - Determines whether or not argparser exits with error

Reviewer: ```suggestion
   * exit_on_error_ - Determines whether or not argparser exits with an error
```

>    * exit_on_error_ - Determines whether or not argparser exits with error info

Reviewer 2: argparser -> ArgumentParser

Author: Thanks, Stéphane. done.

>      when an error occur. (default: ``True``)

Reviewer: ```suggestion
     when an error occurs. (default: ``True``)
```

>      error info when an error occur. (default: ``True``)

Reviewer 2: when an error occur**s**

Author: ops, thank you :)

     +h, ++help  show this help message and exit

exit_on_error
^^^^^^^^^^^^^

> Normally, when you pass an wrong argument list to the :meth:`~ArgumentParser.parse_args`

Reviewer: A basic grammar correction would be to just change "an" to "a", but I think that "invalid" makes more sense in this context than "wrong":

```suggestion
Normally, when you pass an invalid argument list to the :meth:`~ArgumentParser.parse_args`
```

method of an :class:`ArgumentParser`, it will exit with error info.

> If user would like catch error by themselves, they can enable this feature by setting

Reviewer: Minor grammar fix and "by themselves" to "manually":

```suggestion
If the user would like catch errors manually, they can enable this feature by setting
```

> If the user would like catch errors manually, they can enable this feature by setting

Reviewer 2: If the user would like catch errors manually, the feature can be enable by setting
